# HG changeset patch
# User sgehwolf
# Date 1468778064 -3600
#      Sun Jul 17 18:54:24 2016 +0100
# Node ID 6000574e256d84b2bab5cb9f0e2e2fa4ffa587f3
# Parent  bb14c5df70880af9e947b5a7b4b258b62d23931a
S8078504, PR3094: Zero fails to build
Reviewed-by: kvn, simonis

diff --git openjdk.orig/hotspot/src/cpu/zero/vm/vm_version_zero.hpp openjdk/hotspot/src/cpu/zero/vm/vm_version_zero.hpp
--- openjdk.orig/hotspot/src/cpu/zero/vm/vm_version_zero.hpp
+++ openjdk/hotspot/src/cpu/zero/vm/vm_version_zero.hpp
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2003, 2010, Oracle and/or its affiliates. All rights reserved.
- * Copyright 2007 Red Hat, Inc.
+ * Copyright 2015 Red Hat, Inc.
  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
  *
  * This code is free software; you can redistribute it and/or modify it
@@ -34,6 +34,7 @@
   static const char* cpu_features() {
     return "";
   }
+  static void initialize();
 };
 
 #endif // CPU_ZERO_VM_VM_VERSION_ZERO_HPP
